integration_tests/sdk/data_integration_tests/athena_test.py

```diff
@@ -0,0 +1,48 @@
+import pandas as pd
+import pytest
+from aqueduct.error import InvalidUserActionException
+from aqueduct.integrations.sql_integration import RelationalDBIntegration
+
+from aqueduct import LoadUpdateMode, op
+
+from ..shared.naming import generate_table_name
+from .save import save
+
+
+@pytest.fixture(autouse=True)
+def assert_data_integration_is_relational(client, data_integration):
+    assert isinstance(data_integration, RelationalDBIntegration)
+
+
+def test_athena_integration_table_retrieval(client, data_integration):
+    df = data_integration.table(name="hotel_reviews")
+    assert len(df) == 100
+    assert list(df) == [
+        "hotel_name",
+        "review_date",
+        "reviewer_nationality",
+        "review",
+    ]
+
```

Comment:
Based on a quick skim through relational_test.py, I think we're just missing the case where we parameterize sql queries here? Can you also add that?

Reply:
added